### PR TITLE
Clean up the application paths for Rhino/GH

### PR DESCRIPTION
**Pull request by tomvanmele**
### What type of change is this?

- [ ] Bug fix in a **backwards-compatible** manner.
- [ ] New feature in a **backwards-compatible** manner.
- [ ] Breaking change: bug fix or new feature that involve incompatible API changes.
- [ ] Other (e.g. doc update, configuration, etc)

### Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] I added a line to the `CHANGELOG.md` file in the `Unreleased` section under the most fitting heading (e.g. `Added`, `Changed`, `Removed`).
- [ ] I ran all tests on my computer and it's all green (i.e. `invoke test`).
- [ ] I ran lint on my computer and there are no errors (i.e. `invoke lint`).
- [ ] I added new functions/classes and made them available on a second-level import, e.g. `compas.datastructures.Mesh`.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: src/compas_rhino/__init__.py [hunk 1]
```diff
@@ -60,172 +92,223 @@ def _check_rhino_version(version):
...
-        python_plugins_path = _get_python_plugins_path_win32(version)
-
+        app = os.path.join(os.getenv('ProgramFiles'), 'Rhino {}'.format(version))
     elif compas.OSX:
-        python_plugins_path = _get_python_plugins_path_mac(version)
+        paths = {
+            '5': ['/', 'Applications', 'Rhinoceros.app'],
+            '6': ['/', 'Applications', 'Rhinoceros.app'],
+            '7': ['/', 'Applications', 'Rhino 7.app', ],
+            '8': ['/', 'Applications', 'RhinoWIP.app', ]
```
Comment (gonzalocasas) [inline src/compas_rhino/__init__.py:150]: I think this part will need a re-work at some point. It probably needs a more elaborate check, like, if we're trying to install on `7`. We could search for `['Rhinoceros.app', `Rhino 7.app`, `Rhino7.app`] and maybe others, and then check in the metadata in that folder to make sure it really is what we want.

**Review by gonzalocasas — APPROVED** [review]
Generally LGTM, but I didn't have time to go deep into some of the details of the change. Do you want to get this merged and included in today's release?

**Comment by tomvanmele** [comment]
not necessarily, but we could. as far as i can tell (and intended), it just does what already was happening before, but just in a much more readable and cleaner way...

File: src/compas_rhino/__init__.py (hunk 1 above)
Comment (tomvanmele) [reply src/compas_rhino/__init__.py:150]: yes, absolutely. but just to be clear, i haven't changed the functionality (also not this part) just cleaned it up...

**Comment by tomvanmele** [comment]
i leave it up to you, considering the workshop on monday. there is still some time for testing though. i can install all possible versions on my windows laptop and on parallels to check if it still behaves as before...

**Comment by gonzalocasas** [comment]
Let's go for it then...! 
I can do the testing on Windows, and you do the mac test (which you probably already did).

File: src/compas_rhino/__init__.py (hunk 1 above)
Comment (gonzalocasas) [reply src/compas_rhino/__init__.py:150]: yep yep, i know... it's just that every time I read it, i remember "uh, we should change this"

**Pull request merged**